plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterCheckboxList.php

```diff
@@ -86,25 +79,32 @@ class="wc-block-product-filter-checkbox-list__item"
 					>
 						<label
 							class="wc-block-product-filter-checkbox-list__label"
-							for="<?php echo esc_attr( $item['id'] ); ?>"
+							for="<?php echo esc_attr( $item_id ); ?>"
 						>
 							<span class="wc-block-product-filter-checkbox-list__input-wrapper">
 								<input
-									id="<?php echo esc_attr( $item['id'] ); ?>"
+									id="<?php echo esc_attr( $item_id ); ?>"
 									class="wc-block-product-filter-checkbox-list__input"
 									type="checkbox"
-									aria-label="<?php echo esc_attr( $aria_label ); ?>"
-									data-wp-on--change--parent-action="<?php echo esc_attr( $parent . '::actions.toggleFilter' ); ?>"
+									aria-label="<?php echo esc_attr( $item['ariaLabel'] ); ?>"
+									data-wp-on--change="actions.toggleFilter"
 									value="<?php echo esc_attr( $item['value'] ); ?>"
-									data-wp-bind--checked="<?php echo esc_attr( $parent . '::state.isItemSelected' ); ?>"
-									data-filter-item="<?php echo esc_attr( wp_json_encode( $item, JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_QUOT | JSON_HEX_AMP ) ); ?>"
+									data-wp-bind--checked="state.isFilterSelected"
+									data-wp-context="<?php echo esc_attr( wp_json_encode( array( 'item' => $item ), JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_QUOT | JSON_HEX_AMP ) ); ?>"
```

Comment:
nitpick, I wonder if we should write a wrapper/helper for `escape_attr` that passes all the JSON encoding attributes. this gets ridiculously noisy in HTML rendering. Doesn't have to be done in this PR though of course.

Reply:
@luisherranz looks like there isn't a function that only returns data, not the whole attribute string. Do you think we should add one, to use for `get_block_wrapper_attributes`, for example?